handler.go

```diff
@@ -453,15 +453,17 @@ func (handler *Handler) sendError(w http.ResponseWriter, r *http.Request, err er
 		status = 500
 	}
 
-	reason := err.Error()
+	reason := err.Error() + "\n"
 	if r.Method == "HEAD" {
 		reason = ""
 	}
 
-	w.Header().Set("Content-Type", "text/plain")
+	//https://golang.org/src/net/http/server.go#L1429
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
```

Comment:
Great addition, but what about adding this to all responses and not just in case of an error?